Api.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
```

Comment:
I am very nervous about scripts in root folder 😂 

"scripts" (whichever language they are written in) that are supporting the toolchain of the current project should please go into `scripts/` folder.

Reply:
Well, a simple script `sami.php` which is _lowercase_ seems ok to be in root. This file is - by it's case-  not looking like a script and instead gives the impression, this file should be moved into `src/`. The repo shouldn't give the look something is placed wrongly as this causes then confusion.

Reply:
Well, to be honest `composer api` script makes sense for me while installing sami via composer.
Imho we should completely remove it. Of course, we'll leave `sami.php` to configure the api tool.
What do you think about?

Reply:
jap, `sami.php` for configuration I'm fine with. I'm also fine with a script to do all the stuff and be called by `composer api`. I just don't like this script to be in the root of the repo - that's what I meant.

Reply:
Done.
Now the script lives in `scripts/` directory and the namespace (only --dev) is `gossi\\docblock\\scripts\\`.